app/src/androidTest/java/ch/hikemate/app/ui/map/MapScreenTest.kt

```diff
@@ -0,0 +1,58 @@
+package ch.hikemate.app.ui.map
```

Comment:
Request: Please let me know once you've pushed the test so that I can review them!

Reply:
The tests that are currently implemented will be the only ones for this PR. I will implement most of the other ones when working on #41, because I need to perform some modification of how the view model works before testing the map screen.

I can ask you for a review on the other PR once it will be ready though.

If that is OK to you, we can mark this conversation as resolved and I'll request your review on the new PR I'll create for #41.